lessons/python/data_analysis/lesson.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: page
+title: 'Data anlysis in Python'
+visible: true
+tags:
+  - python
+  - intermediate
+---
+
+ - **Authors**: Ahmed Hasan, James Santangelo, borrowing from [UofT Coders lesson by Luke Johnston](https://github.com/UofTCoders/studyGroup/blob/gh-pages/lessons/python/pandas2/UofT-pandas.ipynb)
+ - **Research field**: Biology
+ - **Lesson topic**: Introduction to Data analysis in Python
+ - **Lesson content URL**: <https://github.com/utm-coders/studyGroup/tree/gh-pages/lessons/python/data_analysis>
+
+## Preface: ##
+
+This is an introduction to data analysis in Python, including data manipulation and
+visualisation, and an introduction to common Python module in scientific computation.
+
+We will be walking through the lesson content together using a
+Jupyter notebook. We highly recommend installing [Jupyter lab](https://jupyterlab.readthedocs.io/en/stable/getting_started/installation.html)
+through [Anaconda](https://www.anaconda.com/distribution/). Detailed
+download instructions of all required software can be found [HERE](https://github.com/utm-coders/studyGroup/blob/gh-pages/lessons/install-git-python-r.md)
+
+
+## Lesson Overview: ##
+
+The lesson covers:
+
+* Python class methods and attributes
+* The Numpy module for scientific computation
```

Comment:
Very minor edit but package names might look better formatted as code - ie `numpy` and `pandas` instead of Numpy and Pandas